aptos-move/aptos-validator-interface/src/lib.rs

```diff
@@ -117,6 +146,7 @@ pub struct DebuggerStateView {
     query_sender:
         Mutex<UnboundedSender<(StateKey, Version, std::sync::mpsc::Sender<Option<Vec<u8>>>)>>,
     version: Version,
+    pub data_read_stake_keys: Option<Arc<Mutex<HashMap<StateKey, StateValue>>>>,
```

Comment:
stake or state?

Reply:
Also might be better to keep it private and add a function to dump the hashmap directly.

Reply:
done